addons/project/models/project_task_type.py

```diff
@@ -109,38 +167,12 @@ def _compute_disabled_rating_warning(self):
             else:
                 stage.disabled_rating_warning = False
 
+    @api.depends('project_ids')
+    def _compute_user_id(self):
+        # If project_ids is set after stage creation (e.g. when setting demo data), the default user_id has to be removed
+        self.sudo().filtered('project_ids').user_id = False
```

Comment:
> FYI, not sure how, but I remember talking recently with Alexandra about stages that have both a user and projects causing issues. Could you check with her that the fix wouldn't interfere with this?

Normally this should be avoided by a constraint, but I'll check with her :wink:

Reply:
> FYI, not sure how, but I remember talking recently with Alexandra about stages that have both a user and projects causing issues. Could you check with her that the fix wouldn't interfere with this?

I've discussed it with her, it was an old problem that should not be present since we have this constraint now :+1: